src/Turbo/README.md

```diff
@@ -288,8 +289,8 @@ class ChatController extends AbstractController
     <div id="messages" {{ turbo_stream_listen('chat') }}>
         {#
             The messages will be displayed here.
-            "turbo_stream_listen()" automatically registers a Stimulus controller that subscribes to the "chat" Mercure topic using EventSource.
-            The connection to the Mercure Hub is automatically closed when this HTML block is removed.
+            "turbo_stream_listen()" automatically registers a Stimulus controller that subscribes to the "chat" topic of the transport.
+            The transport connection is automatically closed when this HTML block is removed.
```

Comment:
Same here, it's transport-specific.

Reply:
I think we should remove this line actually, it's too deep into the details.